components/support/nimbus-cli/src/cmd.rs

```diff
@@ -118,7 +123,7 @@ impl LaunchableApp {
 
     fn unenroll_all(&self) -> Result<bool> {
         let payload = json! {{ "data": [] }};
-        self.start_app(false, Some(&payload), true)
+        self.start_app(false, Some(&payload), true, None)
```

Comment:
Do we need this `None` here since it's optional?

Reply:
Yep, we do still need to pass _something_, even if it's a `None` `Option`.

Rust doesn't do default parameter values or method overloading and as far as I've been able to determine from research there aren't plans to support either.